ribs/emitters/_gaussian_emitter.py

```diff
@@ -52,6 +53,7 @@ def __init__(self,
         self._x0 = np.array(x0, dtype=archive.dtype)
         self._sigma0 = archive.dtype(sigma0) if isinstance(
             sigma0, (float, np.floating)) else np.array(sigma0)
+        self._sigma = sigma0 if sigma is None else sigma
```

Comment:
It looks like we allowed `sigma0` to be an array-like; can you make `sigma` also be an array-like?

Reply:
The reason we want array-like is that you may not want to have isotropic noise; you may want to have different variance in each dimension.